dev/com.ibm.ws.collector.manager/src/com/ibm/ws/collector/manager/internal/CollectorManagerImpl.java

```diff
@@ -370,8 +370,10 @@ private synchronized void startSourceWithBufferManager(String sourceId, Handler
         /*
          * An active Buffer already exists. This must mean we subscribed two handlers wanting
          * the same source really quickly back-to-back before the source was set into CollectorManager
+         *
+         * Also if it is audit, there is no point of creating a bufferManager here, it won't start up the audit source.
          */
-        if (activeBuffMgrServices.containsKey(sourceId)) {
+        if (activeBuffMgrServices.containsKey(sourceId) || sourceId.contains(CollectorConstants.AUDIT_LOG_SOURCE)) {
```

Comment:
should audit have a separate Source class that gets started when someone subscribes?  not clear why audit is different here.

Reply:
Audit team did not update their code to start their sources the OSGI way where a BufferManager service is created with distinct properties set (i.e the sourceID of the source we want to start) which will in turn cause a targeted source to start up.

They are still relying on the _old_ behavior where CollectorManagerImpl would create a BufferManager for them and is passed on to their Sources and Handlers.

Reply:
please open an issue for audit team to address that as a tech debt item.  would be good to avoid any audit-specific things in CollectorManagerImpl